public/app/screens/main/navigation/navigation.jsx

```diff
@@ -26,6 +26,7 @@ var Navigation = React.createClass({
 
     }.bind(this));
 
+    // Adjust the two-up class according to Foundation icon bar conventions (three-up, four-up, ...)
```

Comment:
Mark comment with `TODO:` as we need to map `items.length` to the correct CSS class name